apps/site/src/app/about/TeamCard.tsx

```diff
@@ -16,18 +22,19 @@ import styles from "./TeamCard.module.scss";
 //
 
 //
+
 interface TeamCardProps {
 	name: string;
 	position: string;
-	image: string;
-	linkedInUrl: string;
+	image: z.infer<typeof SanityImageReference> | null;
+	linkedInUrl?: string;
 }
 const TeamCard = ({ name, position, image, linkedInUrl }: TeamCardProps) => {
 	return (
 		<div className={styles.teamCard}>
 			<div className={styles.imagesContainer}>
 				<img
-					src={image || blank.src}
+					src={image ? urlBuilder.image(image).url() : blank.src}
```

Comment:
`urlBuilder.image(image).size(200, 200).dpr(3).url()`

.size will scale the image and dpr will set the resolution to 3x the crop size